doc/install.txt

```diff
@@ -69,818 +48,44 @@ The following libraries and software are optional:
-
-    .. code-block:: cfg
-
-      [gcc]
-      cxxflags = -L/opt/local/lib
-
-- An obscure ``Bus error`` can sometimes be caused when linking
-  Theano-generated object files against the ``framework`` library in Leopard.
-  For this reason, we have disabled linking with ``-framework Python``, since on
-  most configurations this solves the ``Bus error`` problem. If this default
-  configuration causes problems with your Python/Theano installation and you think
-  that linking with ``-framework Python`` might help, then either set
-  the :envvar:`THEANO_FLAGS` environment variable with
-  ``THEANO_FLAGS=cmodule.mac_framework_link`` or edit your ``~/.theanorc`` to
-  contain
-
-    .. code-block:: cfg
-
-      [cmodule]
-      mac_framework_link=True
-
-- More generally, to investigate libraries issues, you can use the ``otool -L``
-  command on ``.so`` files found under your ``~/.theano`` directory. This will
-  list shared libraries dependencies, and may help identify incompatibilities.
-
-Please inform us if you have trouble installing and running Theano on your Mac.
-We would be especially interested in dependencies that we missed listing,
-alternate installation steps, GPU instructions, as well as tests that fail on
-your platform (use the ``theano-users@googlegroups.com`` mailing list, but
-note that you must first register to it, by going to `theano-users`_).
-
-
-Windows
--------
-
-:ref:`install_windows` provides step-by-step instructions on how to install Theano on 32- or 64-bit Windows systems, using freely available
-tools and compilers.
-
-Editing code in Visual Studio
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-You will find a Visual Studio solution file (``Theano.sln``) in the root of
-the Theano repository. Note that this project file may not be kept up-to-date
-and is not officially supported by the core Theano developers: it is provided
-for convenience only.
-Also, be aware that it will not make Theano use Visual Studio to compile C
-files: it is only meant to provide an easy way to edit Theano code within
-the Visual Studio editor.
-
-
-Windows Installation References
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-1. http://stackoverflow.com/questions/9047072/windows-python-version-and-vc-redistributable-version
-2. http://stackoverflow.com/questions/1865069/how-to-compile-a-64-bit-application-using-visual-c-2010-express
-3. http://blog.victorjabur.com/2011/06/05/compiling-python-2-7-modules-on-windows-32-and-64-using-msvc-2008-express/
-4. http://stackoverflow.com/questions/126279/c99-stdint-h-header-and-ms-visual-studio
-5. http://stackoverflow.com/questions/11182765/how-can-i-build-my-c-extensions-with-mingw-w64-in-python
-6. https://mail.python.org/pipermail/python-announce-list/2014-September/010457.html
-
-
-Generating the documentation
-----------------------------
+We recommend you look at the documentation on the website, since it
+will be more current than the documentation included with the package.
```

Comment:
I wanted to take it off from that section because I agree it doesn't belong there. The only place that would make sense is here http://deeplearning.net/software/theano/#documentation however wouldn't a code snippet be awkward on that page?